lpa-submissions-e2e-tests/cypress/integration/appealsInImmediateArea/appealsInImmediateAreaSteps.js

```diff
@@ -21,7 +21,7 @@ When(`the user selects Save and Continue`, () => {
 Then(
   `the user is presented with the page 'Tell us about any appeals in the immediate area'`,
   () => {
-    cy.verifyAppealsInImmediateAreaCaption();
+    cy.verifyAboutAppealCaption();
```

Comment:
Can we also check the text of the question?

Reply:
Happy to add this to the step file for this ticket but think it's outside the scope of the ticket to change for the immediate area question